src/System.Security.Cryptography.Algorithms/src/ApiCompatBaseline.net463.txt

```diff
@@ -3,6 +3,8 @@ MembersMustExist : Member 'System.Security.Cryptography.ECDsa.Create(System.Secu
 MembersMustExist : Member 'System.Security.Cryptography.ECDsa.Create(System.Security.Cryptography.ECParameters)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Security.Cryptography.ECDsa.ExportExplicitParameters(System.Boolean)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Security.Cryptography.ECDsa.ExportParameters(System.Boolean)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'System.Security.Cryptography.ECDsa.FromXmlString(System.String, System.Security.Cryptography.ECKeyXmlFormat)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'System.Security.Cryptography.ECDsa.ToXmlString(System.Security.Cryptography.ECKeyXmlFormat)' does not exist in the implementation but it does exist in the contract.
```

Comment:
I don't think I'd bother adding them to ECDsaOpenSsl without adding them to the base class.  But that would be net-new API (either for ECDsaOpenSsl or for ECDsa); so that'd be an api review thing.